### PR TITLE
Put links back on one line

### PR DESCRIPTION
**Pull request by nottrobin**

### CODE REVIEW

File: data/en_.md [hunk 1]
```diff
@@ -11,16 +11,14 @@ apps with stunning animated and fluid UIs.
...
 
-![Make apps with style](https://assets.ubuntu.com/sites/ubuntu/latest/u/img/ho
-mepage/morph/phone-screen.jpg)
+![Make apps with style](https://assets.ubuntu.com/sites/ubuntu/latest/u/img/homepage/morph/phone-screen.jpg)
 
 
 
 
 
 ![Bring your content to the frontpage](https://developer.ubuntu.com/static/dev
```
Comment (pmahnke) [inline data/en_.md]: this doesn't look right, do you need to run it a second time @nottrobin

File: data/en_blog_2015_.md [hunk 2]
```diff
@@ -53,34 +45,26 @@ Nov. 13, 2015
...
 
-#  [Using Eclipse IOT 101 : MQTT will get you started on your IoT
-journey](/en/blog/2015/11/12/using-eclipse-iot-101-mqtt-will-get-you-started-
-your-iot-journey/)
+#  [Using Eclipse IOT 101 : MQTT will get you started on your IoTjourney](/en/blog/2015/11/12/using-eclipse-iot-101-mqtt-will-get-you-started-your-iot-journey/)
 
 ![](/static/devportal_uploaded/5f64a2b7-6fa9-4cab-a315-837cdc8bf611-3e22b70a-d
-372-464d-b279-ee92a2e5ce53-media/2015/11/12/paho_logo_400.png)The [Eclipse
-Foundation](http://eclipse.org) has become a new home for a number of IoT
+372-464d-b279-ee92a2e5ce53-media/2015/11/12/paho_logo_400.png)The [EclipseFoundation](http://eclipse.org) has become a new home for a number of IoT
```
Comment (pmahnke) [inline data/en_blog_2015_.md]: another

File: data/en_blog_2015_.md [hunk 3]
```diff
@@ -196,31 +173,24 @@ July 23, 2015
...
-#  [Celebrate Ubuntu - Beijing Hackathon](/en/blog/2015/07/14/celebrate-
-ubuntu-beijing-hackathon-sum/)
+#  [Celebrate Ubuntu - Beijing Hackathon](/en/blog/2015/07/14/celebrate-ubuntu-beijing-hackathon-sum/)
 
-![Ubuntu设计-
-ol-01.jpg](https://lh3.googleusercontent.com/_ZZ0piCQbas7eFCQHDeHUCTDTXUjpf-Mu
-TnsnMWCUeORzrE6M5nJHG7iJ1ChHz0kGK2bDaVIaHczp0kELbNsiRrDxiROPU6A_dbiocCn6dZQdbO
--RyWsXDAWS7iNbRD_vvit2tI)
+![Ubuntu设计-ol-01.jpg](https://lh3.googleusercontent.com/_ZZ0piCQbas7eFCQHDeHUCTDTXUjpf-Mu
+TnsnMWCUeORzrE6M5nJHG7iJ1ChHz0kGK2bDaVIaHczp0kELbNsiRrDxiROPU6A_dbiocCn6dZQdbO-RyWsXDAWS7iNbRD_vvit2tI)
```
Comment (pmahnke) [inline data/en_blog_2015_.md]: another

**Review by pmahnke — CHANGES_REQUESTED** [review]
looks like you might need to run that script twice?

**Comment by nottrobin** [comment]
@pmahnke fixed, I hope?

**Review by pmahnke — APPROVED** [review]
LGTM 👍

**Comment by nottrobin** [comment]
I think this is redundant as the code now live in [ubuntudesign/phone-docs](https://github.com/ubuntudesign/phone-docs). Closing.

**Pull request closed without merging**